src/unlimited.cpp

```diff
@@ -429,10 +408,6 @@ void UnlimitedSetup(void)
         mallocedStats.push_front(new CStatHistory<uint64_t>("net/send/msg/" + *i));
     }
 
-    xpeditedBlk.reserve(256);
```

Comment:
Thanks for the review.

reserve() exists as an optimization to get away with a single allocation when you know with high probability how many entries you want.

We can only avoid a lock by assuming any preallocated size is enough, which is not something we should do in production code (if we want to do that we could just use a C-style array).  Adding/removing entries to any of these vectors happens very rarely and should not be the focus of our optimization efforts IMO.

Point taken though.